### PR TITLE
Fix building with PCH disabled.

### PR DESCRIPTION
**Pull request by rohit-n**

### CODE REVIEW

**Comment by BhaaLseN** [comment]
I don't think the point was adding Common.h back to all of these...and that one string.h include looks off too.

File: Source/Core/AudioCommon/Mixer.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 #include <atomic>
 #include <mutex>
-#include <string>
+#include <string.h>
```
Comment (lioncash) [inline Source/Core/AudioCommon/Mixer.h]: \<string\> is necessary. If you need the other one, include \<cstring\>

File: Source/Core/Core/HW/BBA-TAP/TAP_Unix.cpp [hunk 2]
```diff
@@ -2,6 +2,8 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <string.h>
```
Comment (lioncash) [inline Source/Core/Core/HW/BBA-TAP/TAP_Unix.cpp]: cstring

File: Source/Core/Core/HW/GCMemcard.cpp [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 #include <algorithm>
 #include <cinttypes>
-#include <string>
+#include <string.h>
```
Comment (lioncash) [inline Source/Core/Core/HW/GCMemcard.cpp]: \<string\> is necessary. If you need the other one, include \<cstring\>

**Comment by lioncash** [comment]
> I don't think the point was adding Common.h back to all of these...and that one string.h include looks off too.

Many are likely replaceable with cstring or cstdlib, there are some files that rely on the __forceinline definition though.

**Comment by rohit-n** [comment]
I included Common/Common.h mostly for _trans, UNUSED, and scm_rev_git_str.

**Comment by Parlane** [comment]
@dolphin-emu-bot rebuild

**Comment by Parlane** [comment]
@dolphin-emu-bot rebuild

**Comment by Parlane** [comment]
@dolphin-emu-bot rebuild

**Comment by Parlane** [comment]
@dolphin-emu-bot rebuild

**Comment by waddlesplash** [comment]
LGTM

**Pull request merged**